discord/guild.py

```diff
@@ -1112,6 +1112,55 @@ def _create_channel(self, name, overwrites, channel_type, category=None, **optio
         fields['system_channel_flags'] = system_channel_flags.value
         await http.edit_guild(self.id, reason=reason, **fields)
 
+    def fetch_members(self, *, limit=1, after=None, oldest_first=None):
+        """|coro|
+
+        Retrieves an :class:`.AsyncIterator` that enables receiving the guild's members.
+
+        .. note::
+
+            This method is an API call. For general usage, consider :attr:`members` instead.
+
+        .. versionadded:: 1.2.0
+
+        All parameters are optional.
+
+        Parameters
+        ----------
+        limit: Optional[:class:`int`]
+            The number of members to retrieve.
+            Defaults to 1.
+        after: Optional[Union[:class:`.abc.Snowflake`, :class:`datetime.datetime`]]
+            Retrieve members after this date or object.
+            If a date is provided it must be a timezone-naive datetime representing UTC time.
+        oldest_first: Optional[:class:`bool`]
```

Comment:
I honestly don't think this makes sense for this context since the endpoint is ordered by snowflake (and not by join date) so it might surprise members and cause woes when it comes to support. Might as well just remove the `oldest_first` logic entirely here.